management/proto/management.proto

```diff
@@ -59,6 +63,47 @@ message ServerKeyResponse {
   google.protobuf.Timestamp expiresAt = 2;
 }
 
-message Empty {
+message Empty {}
+
+// WiretrusteeConfig is a common configuration of any Wiretrustee peer. It contains STUN, TURN, Signal and Management servers configurations
+message WiretrusteeConfig {
+  // a list of STUN servers
+  repeated HostConfig stuns = 1;
+  // a list of TURN servers
+  repeated HostConfig turns = 2;
+
+  // a Signal server config
+  HostConfig signal = 3;
+
+  // a Management server config
```

Comment:
Is it needed? Since client was able to request this data, it should know where management server lives

Reply:
In case we will need to change it.
Not likely to happen, also we could change via DNS.
I can remove it